tests/pathing.py

```diff
@@ -480,27 +479,25 @@ def calc_roll_angle(y_velocity: float) -> float:
             0.02  # Currently a constant, should be calculated with a PID loop
         )
 
-        # Gets next gate position and orientation
-        gate_pose = reproducer.get_gate_pose(next_gate)
-        gate_orientation = airsimdroneracinglab.utils.to_eularian_angles(gate_pose.orientation)
+        # Gets next gate position
+        gate_pose: airsimdroneracinglab.Pose = reproducer.get_gate_pose(next_gate)
```

Comment:
Is the drone orientation no longer needed?

Reply:
It was originally used in pathing calculations and for debugging, but after breaking many of the calculations into specific functions it is no longer needed.